grizli/tests/test_autoscript.py

```diff
@@ -5,8 +5,8 @@
 
 import numpy as np
 
-from .. import utils
-from ..pipeline import auto_script
+from grizli import utils
+from grizli.pipeline import auto_script
```

Comment:
Why did you switch all the tests to absolute imports? Because the installation now lives in site-packages?

Reply:
Absolute imports is best practice for `pytest` testing.

And yes, its important to make sure that packaging worked - i.e. so you can test against the installed version of the code in `site-packages`.  So now (after a couple fixes):

```
$ pip install .
$ cd
$ pytest --pyargs grizli
```

works.  That runs tests against the installed version of the code and it guarantees that all the files needed by the tests are actually installed into `site-packages`.